data/constants.json

```
@@ -19,6 +19,7 @@
   "itemBreakChance": 0.5,
   "brokenItemRepairPenaltySeconds": 300,
   "itemSellCostCoefs": [0.5, 0.0],
+  "itemReforgeCostRecursiveCoefs": [0.0 ,40000.0, 10000.0],

```

That "coefs" thing I have introduced is just complex for no good reason. What about just
```suggestion
  "itemReforgeCosts": [40000, 90000, 150000],
```

---

allows for reforging higher tier weapons.
The price patter i chose come from recursive coefs. I just hid it in the first place by giving directly the values
The patern is 
0 : 0
1 : 40k = 0 +40k
2 : 90k = 40k + 50k
3 : 150k = 90k + 60k
4 : 220k = 150k + 70k
5 : 300k = 220k + 80k

You asked me to keep the code general enough in the upgrade command to allow for rank>3
I thought it was necessary to have reforge do the same as reforging is the inverse of upgrading.

---

The idea was to make the code more generic when it's easy. In that case by using that method we lost a way to quickly check what are the reforge prices but also it adds an algorithm we will have to synchronize in the web ui. So not a big fan.

---

Do you agree with me inputting cost till rank 9 to get a bit of leeway?

---

done